lib/charms/mysql/v0/mysql.py

```diff
@@ -494,7 +494,13 @@ def _on_set_password(self, event: ActionEvent) -> None:
 
     def _get_cluster_status(self, event: ActionEvent) -> None:
         """Action used  to retrieve the cluster status."""
-        if status := self._mysql.get_cluster_status():
+        extended = event.params.get("extended", 0)
```

Comment:
the libpatch on this file should be incremented

Reply:
@arif-ali , I thinks that should be the only pending change.

Reply:
That should be done on the latest push